Makefile

```diff
@@ -22,3 +24,8 @@ uninstall:
 		/lib/modules/*/extra/hid-viewsonic.ko
 	udevadm control --reload
 	depmod -a
+
+distdir = $(PACKAGE)-$(VERSION)
+dist:
+	git archive --format=tar.gz --prefix=$(distdir)/ HEAD > $(distdir).tar.gz
+	git archive --format=zip --prefix=$(distdir)/ HEAD > $(distdir).zip
```

Comment:
We don't really need a .zip, could you remove it?

Also, could you please add a [`.gitattributes` file](https://www.mankier.com/5/gitattributes) marking the `debian` directory with the [`export-ignore` attribute](https://www.mankier.com/5/gitattributes#Effects-Creating_an_archive)? That will help Debian packagers, as they prefer to make their own package description.